### ISSUE TITLE
Discern between Sky, Surface and Depths objects?

### ISSUE THREAD

**Issue by Paperzlel** [zeldamods/objmap-totk#18]
So this one's more of a suggestion but would it be worth having a feature by which an object (such as a weapon in a chest) can only be seen on its respective layer, so that, say, a tree branch on GSI doesn't show up on the surface at Riverside Stable?

**Comment by savage13** [comment]
Something like this. I can make the size of the marker change when the layer switches.

![Screenshot 2023-05-31 at 14-13-23 TotK Object Map](https://github.com/zeldamods/objmap-totk/assets/126514/37547006-3fdf-4763-a75d-c5cdf2af7d46)

**Comment by slaurent22** [comment]
I would prefer if the setting made the depths markers just not show up at all on the surface

**Comment by leoetlino** [comment]
Yeah I think it might be worth having a "search only current layer" checkbox on the search pane.

**Comment by pmbarrett314** [comment]
Even just having different colors/shapes for the marker based on what layer it's on would be super helpful.
